packages/application-generic/src/services/auth/auth.service.ts

```diff
@@ -31,14 +31,15 @@ import { PinoLogger } from '../../logging';
 import { AnalyticsService } from '../analytics.service';
 import { ApiException } from '../../utils/exceptions';
 import { Instrument } from '../../instrumentation';
+import { CreateUser, CreateUserCommand } from '../../usecases/create-user';
```

Comment:
We can't rely in the index exports of the folders because due our usage of App generic is what causes circular dependencies.